geoportailv3/static/js/layerinfo/layerinfoservice.js

```diff
@@ -14,29 +15,91 @@ app.ShowLayerinfo;
 
 
 /**
+ * @param {angularGettext.Catalog} gettextCatalog Gettext catalog.
  * @param {angular.$http} $http Angular $http service
  * @param {angular.$sce} $sce Angular $sce service
  * @param {ngeo.CreatePopup} ngeoCreatePopup Ngeo popup factory service
  * @ngInject
  */
-app.showLayerinfoFactory = function($http, $sce, ngeoCreatePopup) {
+app.showLayerinfoFactory = function(gettextCatalog,
+                                    $http,
+                                    $sce,
+                                    ngeoCreatePopup) {
```

Comment:
Very very minor, but we don't use that type of indentation elsewhere. We just use four-space indentation for function parameters.

```js
app.showLayerInfoFactory = function($http, $sce,
    gettextCatalog, ngeoCreatePopup) {
  // …
```

Also, I'd change the argument order, as done above in my code snippet. Again, this is minor.